htdocs/js/jsx/behavioural_qc_module.jsx

```diff
@@ -0,0 +1,363 @@
+var PagedRowHeader = React.createClass({
```

Comment:
This is in the wrong directory, it should be in the module directory (or jsx directory), we don't put JSX directly in htdocs